src/code_80004740.c

```diff
@@ -103,10 +102,7 @@ void func_80004A1C(animation_type_1 *arg0, s16 *arg1, animation_type_3_triplet a
     func_800048D8(sp4C, sp94, sp8C);
     func_80004740(&gGfxPool->mtxPool[D_8018D120 + 11], sp4C);
     D_80162D7A += 1;
-    // I really don't like the 0x2000000 here, but if it matches, it matches
-    // In decompiled form the mtx pool stuff looks like `&gGfxPool->mtxPool[D_8018D120] + 0x8000002C0`
-    // The `2C0` ought to turn into an 11, like it is a couple lines above. No idea where the 0x2000000 is coming from
-    gSPMatrix(gDisplayListHead++, &gGfxPool->mtxPool[D_8018D120 + 0x2000000 + 11], G_MTX_PUSH | G_MTX_MUL | G_MTX_MODELVIEW);
+    gSPMatrix(gDisplayListHead++, VIRTUAL_TO_PHYSICAL2(&gGfxPool->mtxPool[D_8018D120+ 11]), G_MTX_PUSH | G_MTX_MUL | G_MTX_MODELVIEW);
```

Comment:
`120 + 11` (missing a space)

Reply:
Fix dis one and I'll merge

Reply:
I have some other changes to make if everything else look fine. But yes I'll add the space here too.

Reply:
Sounds good. Let me know when you're done.